src/main/java/com/squad8/s8orangebackend/service/ProjectService.java

```diff
@@ -1,14 +1,35 @@
 package com.squad8.s8orangebackend.service;
 import com.squad8.s8orangebackend.domain.project.Project;
+import com.squad8.s8orangebackend.domain.user.User;
+import com.squad8.s8orangebackend.dtos.ProjectRegistrationDto;
 import com.squad8.s8orangebackend.repository.ProjectRepository;
+import com.squad8.s8orangebackend.repository.UserRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import java.util.List;
 @Service
 public class ProjectService {
     @Autowired
     private ProjectRepository projectRepository;
+    private UserRepository userRepository;
     public List<Project> listProjects() {
         return projectRepository.findAll();
     }
+
+    public Project insertProject(Project project) {
+        return projectRepository.save(project);
```

Comment:
É necessário passar o id do usuário logado. Alteração será realizada quando for implementado o login

Reply:
Ok, vou estudar sobre e implementar posteriormente.